pquery-run.sh

```diff
@@ -173,18 +173,27 @@ diskspace() {
 
 add_handy_scripts(){
   # Add handy stack script
-  cat "${SCRIPT_PWD}/stack.sh" > ${RUNDIR}/${TRIAL}/stack && chmod +x ${RUNDIR}/${TRIAL}/stack
+  if [[ ${MDG} -eq 1 ]]; then
+    SAVE_STACK_LOC=${RUNDIR}/${TRIAL}/node${j}
+  else
+    SAVE_STACK_LOC=${RUNDIR}/${TRIAL}
+  fi
+  cat "${SCRIPT_PWD}/stack.sh" > ${SAVE_STACK_LOC}/stack && chmod +x ${SAVE_STACK_LOC}/stack
```

Comment:
Great solution. Thanks. Remember to do:
```
SAVE_STACK_LOC=
``` 
Directly after this line (defensive coding)